module.nix

```diff
@@ -123,6 +123,29 @@ in
       };
     };
 
+    # Proxy oupost
+    authentik-proxy = {
+      enable = mkEnableOption "authentik Proxy outpost";
+
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
```

Comment:
types.path isn't a good idea for secret files. I fixed the other occurrences in #51, so it's probably the easiest to rebase onto this PR and use pathToSecret.

Reply:
Done!